cli/scorecard/inventory.go

```diff
@@ -52,10 +53,11 @@ func TargetProject(projectID string) Option {
 }
 
 // NewInventory creates a new CAI inventory manager
-func NewInventory(projectID string, bucketName string, options ...Option) (*InventoryConfig, error) {
+func NewInventory(projectID string, bucketName string, caiDirName string, options ...Option) (*InventoryConfig, error) {
```

Comment:
Instead of adding new, potentially empty, params we should probably refactor `bucketName` and `caiDirName` to be Options.

Reply:
addressed in commit https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/314/commits/dfb1e39be1eaff427d9e21f81df5bb8ff87f6352